k8s/perftest/common/divorce/div-cos.yaml

```diff
@@ -28,54 +28,16 @@ spec:
       useInterpodAntiAffinity: true
       applicationInsightsInstrumentKey: "eddbcec4-6ebc-4507-9f22-38d53a92e761"
       image: hmctspublic.azurecr.io/div/cos:prod-135ec6cc
-      ingressHost: div-cos-perftest.service.core-compute-perftest.internal
       environment:
         IDAM_API_URL: "https://idam-api.perftest.platform.hmcts.net"
-        UK_GOV_NOTIFY_API_KEY: "divorce_preprod_live_key-b86e6469-4927-4f50-a029-6ddb523330cf-2409ba37-b5e6-4f03-b052-7d0f0986888d"
-        CASE_MAINTENANCE_SERVICE_API_BASEURL: "http://div-cms-perftest.service.core-compute-perftest.internal"
-        IDAM_CITIZEN_USERNAME: "DivCitizenUser@AAT.com"
-        PAYMENT_API_BASEURL: "http://payment-api-perftest.service.core-compute-perftest.internal"
-        FEATURE_TOGGLE_SERVICE_API_BASEURL: "http://rpe-feature-toggle-api-perftest.service.core-compute-perftest.internal"
-        SEND_LETTER_SERIVCE_BASEURL: "http://rpe-send-letter-service-perftest.service.core-compute-perftest.internal"
-        IDAM_API_REDIRECT_URL: "https://div-pfe-perftest.service.core-compute-perftest.internal/authenticated"
-        IDAM_CASEWORKER_USERNAME: "DivCaseWorkerUser@AAT.com"
-        CASE_FORMATTER_SERVICE_API_BASEURL: "http://div-cfs-perftest.service.core-compute-perftest.internal"
-        SERVICE_AUTH_PROVIDER_URL: "http://rpe-service-auth-provider-perftest.service.core-compute-perftest.internal"
-        CASE_VALIDATION_SERVICE_API_BASEURL: "http://div-vs-perftest.service.core-compute-perftest.internal"
-        DOCUMENT_GENERATOR_SERVICE_API_BASEURL: "http://div-dgs-perftest.service.core-compute-perftest.internal"
-        FEES_AND_PAYMENTS_SERVICE_API_BASEURL: "http://div-fps-perftest.service.core-compute-perftest.internal"
-        MANAGEMENT_ENDPOINT_HEALTH_CACHE_TIMETOLIVE: "30000"
         FEATURE_DN_REFUSAL: "true"
-
-        SPRING_MAIL_HOST: mta.reform.hmcts.net
-        SPRING_MAIL_PORT: 25
-        SPRING_MAIL_TEST_CONNECTION: false
-        SPRING_MAIL_PROPERTIES_MAIL_SMTP_SSL_TRUST: mta.reform.hmcts.net
-        DATAEXTRACTION_STATUS_DA_EMAILTO: da_data_extraction@sharklasers.com
-
-        DIV_SCHEDULER_DB_HOST: "div-cos-perftest.postgres.database.azure.com"
-        DIV_SCHEDULER_DB_PORT: "5432"
-        DIV_SCHEDULER_DB_NAME: "div_scheduler"
-        DIV_SCHEDULER_DB_USER_NAME: "div_scheduler@div-cos-perftest"
-        FLYWAY_URL: "jdbc:postgresql://div-cos-perftest.postgres.database.azure.com:5432/div_scheduler"
-        SCHEDULER_SCHEDULES_CREATE_BULK_CASES_ENABLED: false
-        SCHEDULER_RE_CREATE: true
-        SCHEDULER_ENABLED: true
         FEATURE_RESP_SOLICITOR_DETAILS: "true"
+        SPRING_MAIL_PROPERTIES_MAIL_SMTP_STARTTLS_ENABLE: false
+        SCHEDULER_SEND_UPDATED_CASES_TO_ROBOTICS_ENABLED: true
+        DATAEXTRACTION_STATUS_DA_EMAILTO: da_data_extraction@sharklasers.com
```

Comment:
Does this environment use test data?

Reply:
Yes, it's prod like but fake data

Reply:
OK. This data will end up in a public e-mail inbox.
It doesn't sound like an issue, but I'm just letting you know.